api/scheduler.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import motor
+from beanie import init_beanie
+from rocketry import Rocketry
+from rocketry.conds import daily
+
+from api.config import get_settings
+from api.models.catalog import CoreMetadataDOC, Submission
+
+app = Rocketry(config={"task_execution": "async"})
+
+logger = logging.getLogger()
+
+
+@app.task(daily)
+async def do_daily():
```

Comment:
The scheduler is not needed at this time since all submissions are manually entered instead of sourced from a repository.